src/main/kotlin/anttracker/Request.kt

```diff
@@ -9,84 +10,52 @@ The Request module contains all exported classes and functions pertaining to
 package anttracker.request
 
 import anttracker.contact.enterContactInformation
-import anttracker.contact.getContactInfo
-import anttracker.contact.Name
 import anttracker.contact.selectContact
-import anttracker.db.Issue
-import anttracker.issues.Days
+import anttracker.db.*
 import anttracker.issues.IssueFilter
-import anttracker.issues.IssueId
 import anttracker.issues.enterIssueInformation
 import anttracker.issues.saveIssue
 import anttracker.issues.selectIssue
 import anttracker.product.selectProduct
 import anttracker.product.ProductName
 
 // release imports
-import anttracker.release.ReleaseId
 import anttracker.release.selectRelease
-import java.time.LocalDate
-
-// ----------------------------------------------------------------------------
-// Data class for storing the attributes of a given request.
-// ---
-data class Request(
-    val affectedRelease: ReleaseId,
-    val issue: Issue,
-    val contact: Name,
-    val requestDate: LocalDate,
-)
-
-// any attribute in this may be null - signifying a not being used filtering mechanism
-// only non-null attributes of this class may be considered in the filtering process
-data class RequestFilter(
-    val name: Name?, // by contact name
-    val days: Days?, // since n days
-    val release: ReleaseId?, // by release
-    val issue: IssueId?, // by issue
-)
-
-// -----------------
-
-// ---- database functions
-
-// saves a given request object into the database
-fun saveRequest(
-    request: Request, // in
-) {
-    TODO()
-}
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.time.LocalDateTime
+
+// display a given request to the screen
```

Comment:
I think a more accurate name would be displayRequester(), since it's primarily about the contact info and not the request object per se.
My main question is where is this used, and if the intention is to paginate a list of requesters. If so, then it'd be better to move this function into the printRecord() of a pageOfRequesters subclass.

Reply:
This function is used in enterRequestInformation, after a request is created, displaying the created request. Check line 95. I'm a fan of `displayRequester`!

But yeah, my hope is that it would be re-used in the paginated output in the Issue sub-menu that displays a list of requests for a given issue.

Reply:
Would this be using mine or Eitan's pagination?

Reply:
Up to Eitan! So probably his

Reply:
He might end up not using it tho, so not sure 🤷 

I could just essentially paste the body of the function into where it's used in enterRequestInformation, and let him implement it himself.

@ebarylko - thoughts? Would this function be useful for you?

Reply:
I am not sure. I have not implemented that functionality yet.